scripts/changelog-ci.py

```diff
@@ -1,14 +1,18 @@
 import json
+import yaml
```

Comment:
We should add this with `requests` as it's not a built-in library.

Reply:
I am referring to [PEP 8](https://www.python.org/dev/peps/pep-0008/#imports)

> Imports should be grouped in the following order:
> 
> Standard library imports.
> Related third party imports.
> Local application/library specific imports.
> You should put a blank line between each group of imports.

Reply:
Ah, thank you for explaining! I understand now. I have pushed an updated commit. https://github.com/saadmk11/changelog-ci/pull/46/commits/92be906b9bdd74d149243d7fb9fe33cc7751707e resolves all issues that violate compliance with PEP 8.